src/rdkafka_admin.c

```diff
@@ -5186,10 +5186,10 @@ rd_kafka_AlterConsumerGroupOffsetsResponse_parse(rd_kafka_op_t *rko_req,
         rd_kafka_topic_partition_list_destroy(partitions);
         *rko_resultp = rko_result;
 
-        rd_snprintf(
-            errstr, errstr_size,
-            "AlterConsumerGroupOffset response protocol parse failure: %s",
-            rd_kafka_err2str(reply->rkbuf_err));
+        rd_snprintf(errstr, errstr_size,
```

Comment:
This seems to be a reindent, not sure clang-format will be happy about.

Also, change it so it is conditional and uses the correct error.
```c
if (!err && rkbuf->rkbuf_err)
   err = rkbuf->rkbuf_err;

if (err)
   rd_snprintf(errstr, .....);

return err;
```

Reply:
Did this partly - for now , as :
```
if (rkbuf->rkbuf_err)
   rd_snprintf(errstr, .....);
```

Because `err` should actually be conditionally ignored (handle_OfffsetCommit raises errors for those cases where all partitions have partitions level errors which shouldn't be a global level error for alter offsets)